### PR TITLE
Remove button borders

### PR DESCRIPTION
**Pull request by jsondoo**
**Changes**
- Blue buttons are still blue w/o borders
- Other buttons are grey (same color as the quick start button) w/o borders

- Footer buttons are special case since it is a different background color - @xtinah-w and I discussed with the team and decided on the colors: back button should be black on dark mode, white on light mode (and checked this with accessibility compliance) 

**How To Test**
- check light theme and dark theme

image
image

### CODE REVIEW

**Review by streetcoder123 — DISMISSED** [review]

**Review by jasmingoh — DISMISSED** [review]

File: src/client/src/css/buttonStyles.module.css [hunk 1]
```diff
@@ -1,14 +1,11 @@
 .buttonHighlightedBorder {
```
Comment (jcoc611-microsoft) [inline src/client/src/css/buttonStyles.module.css]: Do we still need this rule? It is the same as `.buttonHighlighted` and it has no border (despite what name suggests)?

Comment (jsondoo) [reply src/client/src/css/buttonStyles.module.css]: good catch, it is the same thing so I replaced it and removed it

**Comment by annielovesjs** [comment]
doo-ing great

**Review by annielovesjs — APPROVED** [review]
doo-ing great

**Review by jcoc611-microsoft — APPROVED** [review]

**Review by ali-kha — APPROVED** [review]

**Pull request merged**